graph/src/main/java/com/kryvapust/graph/files/Graph.java

```diff
@@ -0,0 +1,35 @@
+package com.kryvapust.graph.files;
+
+import lombok.Getter;
+
+import java.util.List;
+
+import static com.kryvapust.graph.files.MainFlow.MAX;
+
+public abstract class Graph {
+    protected final Helper helper;
+    @Getter
+    protected int[][] matrix;
+
+    protected Graph(Helper helper) {
```

Comment:
Helper class looks like a typical util class. So, you can make all Helper methods static and not inject it